packages/database/src/migrations/20200218025613-AddDataGroupsToDataSource.js

```diff
@@ -25,7 +25,7 @@ exports.up = async function(db) {
       JOIN survey ON survey_screen.survey_id = survey.id
       JOIN data_source AS data_group ON survey.code = data_group.code AND data_group.type = 'dataGroup'
       JOIN data_source AS data_element ON question.code = data_element.code AND data_element.type = 'dataElement'
-      WHERE question.type NOT IN ('Instruction', 'SubmissionDate', 'PrimaryEntity')
+      WHERE question.type NOT IN ('Instruction', 'SubmissionDate', 'DateOfData', 'PrimaryEntity')
```

Comment:
This is just so if anyone uses it for reference, they're aware to also ignore DateOfData